static/docs/user-guide/cache-file-linking.md

```diff
@@ -41,61 +40,73 @@ system.
 There are pros and cons to the 3 supported link types: Hard links, Soft or
 Symbolic links, and Reflinks\* in more recent systems. While reflinks bring all
 the benefits and none of the worries, they're not commonly supported in most
-platforms yet. Hard/soft links also optimize speed and space in the file system,
-but carry the risk of breaking your workflow, since updating tracked files in
-the workspace causes data corruption. These 2 link types thus require using
+platforms yet. Hard/soft links optimize speed and space in the file system, but
+carry the risk of breaking your workflow since updating hard/sym-linked files in
+the workspace causes cache corruption. These 2 link types thus require using
 cache protected mode (see the `cache.protected` config option in
-`dvc config cache`). Finally, a 4th "linking" option is to actually `copy` files
-from/to the cache, which is safe but quite inefficient for large files (not
-recommended for GBs of data).
+`dvc config cache`). Finally, a 4th "linking" option is to actually copy files
+from/to the cache, which is safe but inefficient, especially for large files
+(several GBs or more data).
 
-Each file linking option is further detailed below, in order of efficiency:
+File link type benefits summary:
+
+| `cache.type` | speed | space | safe |
+|---|---|---|---|---|
+| `reflink` | x | x | x |
+| `hardlink` | x | x | |
+| `symlink` | x | x | |
+| `copy` | | | x |
+
+Each file linking option is further detailed below, in function of their
+efficiency:
 
 1. **`reflink`** - copy-on-write\* links or "reflinks" are the best possible
-   link type, when available. They're is as fast as hard/symlinks, but don't
-   carry a risk of cache corruption since the file system takes care of copying
-   the file if you try to edit it in place, thus keeping a linked cache file
-   intact.  
-   Unfortunately reflinks are currently supported on a limited number of file
-   systems only (Linux: Btrfs, XFS, OCFS2; MacOS: APFS), but in the future they
-   will be supported by the majority of file systems in use.
+   link type, when available. They're is as efficient as hard/symlinks, but
+   don't carry a risk of cache corruption since the file system takes care of
+   copying the file if you try to edit it in place, thus keeping the linked
+   cache file intact.
+
+   > Unfortunately reflinks are currently supported on a limited number of file
+   > systems only (Linux: Btrfs, XFS, OCFS2; MacOS: APFS), but in the future
+   > they will be supported by the majority of file systems in use.
 
 2. **`hardlink`** - hard links are the most efficient way to link your data to
    cache if both your repo and your cache directory are located on the same file
-   system/drive.  
+   system/ drive.
```

Comment:
remove an extra space here?

Reply:
Maybe I'll change the phrase completely to "same partition or storage device" to be more correct.

Reply:
Done.